docs/changehistory/NextVersion.md

```diff
@@ -7,11 +7,31 @@ publish: false
 Table of contents:
 
 - [NextVersion](#nextversion)
+  - [@itwin/appui-react](#itwinappui-react)
+    - [Popout widgets](#popout-widgets)
+    - [Badge icons](#badge-icons)
+    - [`ThemeManager`](#thememanager)
   - [@itwin/core-react](#itwincore-react)
     - [Changed base `Dialog` component for `MessageBox`](#changed-base-dialog-component-for-messagebox)
-    - [IconWebComponent](#iconwebcomponent)
+    - [`IconWebComponent`](#iconwebcomponent)
+    - [Fixed `ConditionalIconItem` functionality](#fixed-conditionaliconitem-functionality)
   - [@itwin/components-react](#itwincomponents-react)
 
+## @itwin/appui-react
+
+### Popout widgets
+
+- Resolved an issue that prevented opening popouts from within an iFrame. Please note that sandboxed iFrames still require the allow-popups attribute.
+- Eliminated the logic responsible for storing the popped-out widget's location on the screen. Now, when the user closes the widget, it will reappear in the same position it was last closed.
```

Comment:
If you removed the logic, doesn't that mean that the widget **WON'T** reappear in the same location?

Reply:
Good catch - reworded.